contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--2.0.0--2.1.0.sql

```diff
@@ -836,5 +836,72 @@ $$
 LANGUAGE 'pltsql';
 GRANT ALL ON PROCEDURE sys.sp_sproc_columns_100 TO PUBLIC;
  
+CREATE OR REPLACE PROCEDURE sys.sp_statistics(
```

Comment:
we should validate if replacing procedure can fail MVU in any way. If it is used inside a view or similar construct

Reply:
This procedure is not being used in any view or similar construct, I don't think this is going to fail in MVU.